src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,18 @@
+
+
+
+namespace Microsoft.Xunit.Performance.Api
```

Comment:
This is part of the public API so we need to add comments.

Also, can you tell me how you came up with the default values?